huaweicloud/services/elb/resource_huaweicloud_elb_ipgroup.go

```diff
@@ -104,41 +104,41 @@ func resourceIpGroupV3Create(ctx context.Context, d *schema.ResourceData, meta i
 		Name:                d.Get("name").(string),
 		Description:         &desc,
 		IpList:              &ipList,
-		EnterpriseProjectID: config.GetEnterpriseProjectID(d),
+		EnterpriseProjectID: cfg.GetEnterpriseProjectID(d),
 	}
 
 	log.Printf("[DEBUG] Create Options: %#v", createOpts)
```

Comment:
```suggestion
	log.Printf("[DEBUG] Create ELB IP Group options: %#v", createOpts)
```
In my opinion, it is more accurate to write this way.

Reply:
done